documentation/src/main/asciidoc/topics/con_eviction_example.adoc

```diff
@@ -0,0 +1,96 @@
+[id='eviction_example-{context}']
+= Eviction Example
+
+You configure eviction in `infinispan.xml` as part of your cache definition.
+
+.Default memory configuration
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<memory />
+----
+
+This is the default configuration. {brandname} stores cache entries as objects
+in the data container. Eviction is not enabled.
+
+.Passivation without eviction
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<persistence passivation="true">
+  ...
+</persistence>
+
+<memory />
+----
+
+This configuration is not valid and {brandname} writes an error to log files.
+
+Passivation configures {brandname} to write entries to cache stores when it
+evicts those entries. You should always enable eviction if you enable
+passivation.
+
+.Manual eviction
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<memory>
+  <object strategy="MANUAL" />
+</memory>
+----
+
+{brandname} stores cache entries as objects. Eviction is not enabled but
+performed manually using the `evict()` method. In this case, {brandname} does
+not log errors if you enable passivation.
+
+.Object storage with eviction
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<memory>
+  <object size="100" />
+</memory>
+----
+
+{brandname} stores cache entries as objects. Eviction happens when there are
+100 entries in the data container and {brandname} gets a request to create a
+new entry.
+
+.Binary storage with memory-based eviction
+
+[source,xml,options="nowrap",subs=attributes+]
+----
+<memory>
+  <binary size="100" eviction="MEMORY"/>
+</memory>
+----
+
+{brandname} stores cache entries as bytes. Eviction happens when the size of
+the data container reaches 100 bytes and {brandname} gets a request to create a
```

Comment:
Technically it is if a given object would put it over 100 bytes. For example if you had 95 bytes in use and a new object would add another 40, then it would have to evict one or more to make room. MEMORY based eviction is special in that it can actually evict more than one entry for a single put operation.